data_tools/add.py

```diff
@@ -0,0 +1,65 @@
+import firebase_client_wrapper as fcw
+
+import json
+import sys
+
+import validate_code_scheme
+import validate_message_structure
+import validate_user_list
+
+# if (len(sys.argv) != 5):
+#     print ("Usage python add.py crypto_token dataset_id users|schemes|messages path")
+#     print ("add only adds data items that don't currently exist, by id. Existing data")
+#     print ("items remain untouched")
+#     exit(1)
```

Comment:
Any reason as to why this is commented out?

Reply:
Thanks - I had disabled it to do some testing, and had not reanabled it. Fixed now